packages/holocron/src/index.js

```diff
@@ -31,6 +31,7 @@ import {
 import { composeModules } from './ducks/compose';
 import RenderModule from './RenderModule';
 import holocronModule from './publicHolocronModule';
+import forceLoadModule from './loadModule.web';
```

Comment:
This different name from the source file is a bit confusing

Reply:
yeah, `forceLoadModule` doesn't really work internally to this package, since its just the normal loadModule function internally.

This is also not helped by there being an exposed `loadModule` action generator.

I'm all ears for better names, I couldn't come up with any I was happier with.